src/controllers/softtakeover.cpp

```diff
@@ -102,32 +102,74 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
 
     qint64 currentTime = Time::elapsedMsecs();
     // We will get a sudden jump if we don't ignore the first value.
-    if (m_time == 0) {
+    if (m_time == -1) {
         ignore = true;
         // Change the stored time (but keep it far away from the current time)
         //  so this block doesn't run again.
-        m_time = 1;
-//         qDebug() << "SoftTakeover::ignore: ignoring the first value"
-//                  << newParameter;
-    } else if ((currentTime - m_time) > SUBSEQUENT_VALUE_OVERRIDE_TIME_MILLIS) {
```

Comment:
Maybe this is a relict from an earlier version. Now, once we are ins snyc big changes are not ignored even if they arrive after the 50 ms delay has passed.   

It looks like the timer has an effect if we have concurrent control commands. Like automatic pitch changes during you change the pitch from the controller as well. 
If the automatic change happens during a controller slider transaction it is ignored. Without this timer, the controller slider will falls out of sync. If the automatic change happens before or after the manual control, the hardware slider falls out of sync anyway. Is it that what we want? 

If yes, we should add a comment about that.